core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -284,31 +284,36 @@ public SaslConnectionParams getSaslParams() {
     return saslSupplier.get();
   }
 
+  public static BatchWriterConfig getBatchWriterConfig(Properties props) {
```

Comment:
This shouldn't need to be public in order to be unit tested.

```suggestion
  static BatchWriterConfig getBatchWriterConfig(Properties props) {
```